[Doc] update(example model): for OpenAI compatible serving (#4503)
vllm-project/vllm: docs/source/serving/openai_compatible_server.md

You can start the server using Python, or using [Docker](deploying_with_docker.rst):
```bash

> python -m vllm.entrypoints.openai.api_server --model unsloth/llama-3-8b-Instruct --dtype auto --api-key token-abc123

Reviewer: ```suggestion
python -m vllm.entrypoints.openai.api_server --model NousResearch/Meta-Llama-3-8B-Instruct --dtype auto --api-key token-abc123
```

)

completion = client.chat.completions.create(

>   model="unsloth/llama-3-8b-Instruct",

Reviewer: ```suggestion
  model="NousResearch/Meta-Llama-3-8B-Instruct",
```

```python
completion = client.chat.completions.create(

>   model="unsloth/llama-3-8b-Instruct",

Reviewer: ```suggestion
  model="NousResearch/Meta-Llama-3-8B-Instruct",
```

a chat template in its tokenizer configuration. The chat template is a Jinja2 template that
specifies how are roles, messages, and other chat-specific tokens are encoded in the input.

> An example chat template for `unsloth/llama-3-8b-Instruct` can be found [here](https://colab.research.google.com/drive/1Aau3lgPzeZKQ-98h69CCu1UJcvIBLmy2?usp=sharing#scrollTo=vITh0KVJ10qX)

Reviewer: ```suggestion
An example chat template for `NousResearch/Meta-Llama-3-8B-Instruct` can be found [here](https://colab.research.google.com/drive/1Aau3lgPzeZKQ-98h69CCu1UJcvIBLmy2?usp=sharing#scrollTo=vITh0KVJ10qX)
```
